sound/soc/intel/boards/sof_sdw.c

```diff
@@ -295,12 +295,28 @@ static struct sof_sdw_codec_info codec_info_list[] = {
 	},
 	{
 		.part_id = 0x714,
+		.version_id = 3,
+		.direction = {false, true},
+		.dai_name = "rt715-aif2",
```

Comment:
I see it was like this before, but just double-checking: the "715" DAI name is correct for 714 too?

Reply:
> I see it was like this before, but just double-checking: the "715" DAI name is correct for 714 too?

That is a good question. The dai name comes from codec driver. So, yes, the dai name is "rt715-aif2" for both rt715 and rt714 codecs

Reply:
@bardliao thanks for the explanation! So, ideally that driver should be called "rt714-rt715-aif2"? Not hugely important, and nothing to stop this patch of course.